app/controllers/api/databases.php

```diff
@@ -1250,6 +1258,8 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     ->label('event', 'databases.[databaseId].collections.[collectionId].attributes.[attributeId].create')
     ->label('scope', 'collections.write')
     ->label('audits.resource', 'database/{request.databaseId}/collection/{request.collectionId}')
+    ->label('usage.metric', 'collections.{scope}.requests.update')
+    ->label('usage.params', ['databaseId:{request.databaseId}'])
```

Comment:
These changes are confusing, on 0.16.x this should already be there, is there issue on the `datetime` branch during sync with 0.16.x?

Reply:
Correct, the POST `/attributes/boolean` route was newly added in the datetime branch, before the usage updates were merged into 0.16